shm/tests/NextDayTest.cpp

```diff
@@ -2,32 +2,26 @@
 #include "fruit.hpp"
 #include "gtest/gtest.h"
 #include "item.hpp"
+#include "locators.hpp"
 #include "player.hpp"
 #include "ship.hpp"
 #include "store.hpp"
 #include "time.hpp"
 
 class NextDayTest : public ::testing::Test {
 public:
-    Time time;
-    Player player;
     Alcohol alco;
     Item item;
     Fruit fruit;
     std::vector<std::shared_ptr<Cargo>> test_stock;
-    Store store;
-    Ship ship;
+
     NextDayTest()
-        : time(),
-          player(std::make_unique<Ship>(30, 10, 1, &player, &time), 100),
-          alco("alco", 10, 100, 40),
+        : alco("alco", 10, 100, 40),
           item("item", 5, 50, Item::Rarity::common),
           fruit("fruit", 30, 20, 10),
           test_stock({{std::make_shared<Alcohol>(alco)},
                       {std::make_shared<Item>(item)},
-                      {std::make_shared<Fruit>(fruit)}}),
-          store(test_stock, &time),
-          ship(30, 10, 1, &player, &time) {}
+                      {std::make_shared<Fruit>(fruit)}}) {}
```

Comment:
After you change the TimeServiceLocator class and the time will be set by default, you can leave the nextDayTest as it was before, it works fine then.

Reply:
Done.